setup.py

```diff
@@ -28,6 +28,19 @@
 
 import setuptools
 
+from nemo.package_info import (
+    __contact_emails__,
+    __contact_names__,
+    __description__,
+    __download_url__,
+    __homepage__,
+    __keywords__,
+    __license__,
+    __package_name__,
+    __repository_url__,
+    __version__,
+)
+
```

Comment:
I have no idea how you're not breaking CI with this change. But this change will break setup.py.

Reply:
Fixed.
There is an error in isort: when style scope is just single file, isort doesn't respect the setup.cfg setting for some reason.